### PR TITLE
MDS-2468 Ability to edit equipment and activity tables

### PR DESCRIPTION
**Pull request by JustinMacaulay**
Adds the functionality to add, edit and remove activities and equipment on a NoW.

### CODE REVIEW

File: services/core-web/src/components/noticeOfWork/applications/review/activities/AccessRoads.js [hunk 1]
```diff
@@ -1,69 +1,164 @@
...
+      </div>
+    ),
+  };
+
+  const columns = (isViewMode) =>
+    !isViewMode ? [...standardColumns, removeColumn] : standardColumns;
+
   const transformData = (activities) =>
-    activities.map((activity) => ({
-      type: activity.activity_type_description || Strings.EMPTY_FIELD,
```
Comment (pdittaro) [inline services/core-web/src/components/noticeOfWork/applications/review/activities/AccessRoads.js]: Have we move away from using Strings.EMPTY_FIELD?

File: services/core-web/src/components/noticeOfWork/applications/review/activities/Equipment.js [hunk 2]
```diff
@@ -1,43 +1,126 @@
...
   equipment: [],
 };
 
 export const Equipment = (props) => {
-  const columns = [
+  const targetActivity = `${props.activity}.equipment`;
+
+  const editEquipment = (event, rowIndex) => {
+    const equipmentToChange = props.equipment[rowIndex];
+    console.log(rowIndex);
```
Comment (Jsyro) [inline services/core-web/src/components/noticeOfWork/applications/review/activities/Equipment.js]: debug line.

File: services/core-web/src/components/noticeOfWork/applications/review/activities/AccessRoads.js (hunk 1 above)
Comment (JustinMacaulay) [reply services/core-web/src/components/noticeOfWork/applications/review/activities/AccessRoads.js]: The problem is that if I use them then N/A will be passed to the backend on the form submit in the field.

File: services/core-web/src/components/noticeOfWork/applications/review/activities/Equipment.js (hunk 2 above)
Comment (JustinMacaulay) [reply services/core-web/src/components/noticeOfWork/applications/review/activities/Equipment.js]: Thanks, just removed it.

File: services/core-web/src/components/noticeOfWork/applications/review/activities/AccessRoads.js (hunk 1 above)
Comment (pdittaro) [reply services/core-web/src/components/noticeOfWork/applications/review/activities/AccessRoads.js]: Thats strange behaviour. Is it worth a one-line comment?

**Pull request merged**